setup.py

```diff
@@ -14,10 +13,13 @@
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
 
-P2PD_VERSION = "v0.3.6"
-P2PD_CHECKSUM = "627d0c3b475a29331fdfd1667e828f6d"
```

Comment:
Keeping the checksums for both `p2pd` and `p2p-keygen` would be tiresome. On the other hand, it doesn't hurt if we download these binaries one more time in case if a user reinstalls hivemind.

Reply:
They weigh ~7-8MB each, which is quite a lot for a Python package installation (especially because it happens under the hood). Just curious, how do other Python libraries with significant binary deps (e.g. PyTorch or NumPy) handle this?

Reply:
It does hurt if user reinstalls hivemind in an environment that lacks internet access AKA exactly what we do in the internal testing infrastructure.